assets/js/components/UserMenu.js

```diff
@@ -185,6 +188,18 @@ export default function UserMenu() {
 					aria-controls="user-menu"
 					aria-label={ __( 'Account', 'google-site-kit' ) }
 					tooltip
+					customizedTooltip={
+						<Fragment>
+							<strong>
+								{ __( 'Google Account', 'google-site-kit' ) }
+							</strong>
+							<br />
+							<br />
+							{ userFullName }
+							<br />
+							{ userEmail }
```

Comment:
Two lines breaks is arguably a bit awkward when we could style one to be "double-height", but this is probably fine 😅

Reply:
Yeah I thought using CSS here would be an overkill as we might revisit the popup and tooltip after checking with UX.